Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
I like `SystemInfoGatherer` - it means pretty much the same, it should be unique, and with that we can keep the `SystemInfo` value object naming.